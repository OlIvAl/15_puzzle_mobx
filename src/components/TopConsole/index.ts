import styled from 'styled-components';

const StyledTopConsole = styled.div`
  display: flex;
  justify-content: space-between;
  margin-bottom: 12px;
  width: 100%;
`;

export default StyledTopConsole;