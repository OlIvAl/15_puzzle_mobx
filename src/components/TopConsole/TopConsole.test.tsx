import * as React from 'react';
import TopConsole from '.';
import renderer from 'react-test-renderer';

describe('TopConsole', () => {
  it('он отображается', () => {
    const tree = renderer.create(<TopConsole />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});