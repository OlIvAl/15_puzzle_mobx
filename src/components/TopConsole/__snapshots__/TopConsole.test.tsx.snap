// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TopConsole он отображается 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
  margin-bottom: 12px;
  width: 100%;
}

<div
  className="c0"
/>
`;
